management/storageservice/entities.go

```diff
@@ -12,11 +12,11 @@ type StorageServiceClient struct {
 }
 
 type StorageServiceList struct {
-	XMLName         xml.Name         `xml:"StorageServices"`
-	StorageServices []StorageService `xml:"StorageService"`
+	XMLName         xml.Name                 `xml:"StorageServices"`
```

Comment:
nit: I think this XMLName might not even be necessary, since we're only Unmarshalling this type...